sdk/eventhub/event-hubs/test/public/hubruntime.spec.ts

```diff
@@ -1,191 +1,129 @@
-          const client = clientMap.get(clientType)!;
-          const ids = await client.getPartitionIds({});
-          ids.should.have.members(arrayOfIncreasingNumbersFromZero(ids.length));
-        });
+        hubRuntimeInfo.partitionIds.should.have.members(
+          arrayOfIncreasingNumbersFromZero(hubRuntimeInfo.partitionIds.length),
+        );
+        hubRuntimeInfo.createdOn.should.be.instanceof(Date);
+      });
 
-        it("can be manually traced", async () => {
-          const client = clientMap.get(clientType)!;
-          await assert.supportsTracing(
-            (options) => client.getPartitionIds(options),
-            ["ManagementClient.getEventHubProperties"],
-          );
-        });
+      it.skip("can be manually traced", async function () {
+        const client = clientMap.get(clientType)!;
+        await assert.supportsTracing(
+          (options) => client.getEventHubProperties(options),
+          ["ManagementClient.getEventHubProperties"],
+        );
       });
+    });
 
-      describe(`${clientType}.getEventHubProperties`, () => {
-        it("gets the Event Hub runtime information", async () => {
-          const client = clientMap.get(clientType)!;
-          const hubRuntimeInfo = await client.getEventHubProperties();
-          hubRuntimeInfo.name.should.equal(service.path);
+    describe(`${clientType}.getPartitionProperties`, function () {
+      it("gets the partition runtime information with partitionId as a string", async function () {
+        const client = clientMap.get(clientType)!;
+        const partitionRuntimeInfo = await client.getPartitionProperties("0");
+        partitionRuntimeInfo.partitionId.should.equal("0");
+        partitionRuntimeInfo.eventHubName.should.equal(eventhubName);
+        partitionRuntimeInfo.lastEnqueuedOnUtc.should.be.instanceof(Date);
+        should.exist(partitionRuntimeInfo.lastEnqueuedSequenceNumber);
+        should.exist(partitionRuntimeInfo.lastEnqueuedOffset);
+      });
 
-          hubRuntimeInfo.partitionIds.should.have.members(
-            arrayOfIncreasingNumbersFromZero(hubRuntimeInfo.partitionIds.length),
-          );
-          hubRuntimeInfo.createdOn.should.be.instanceof(Date);
-        });
+      it("gets the partition runtime information with partitionId as a number", async function () {
+        const client = clientMap.get(clientType)!;
+        const partitionRuntimeInfo = await client.getPartitionProperties(0 as any);
+        partitionRuntimeInfo.partitionId.should.equal("0");
+        partitionRuntimeInfo.eventHubName.should.equal(eventhubName);
+        partitionRuntimeInfo.lastEnqueuedOnUtc.should.be.instanceof(Date);
+        should.exist(partitionRuntimeInfo.lastEnqueuedSequenceNumber);
+        should.exist(partitionRuntimeInfo.lastEnqueuedOffset);
+      });
 
-        it("can be manually traced", async function (): Promise<void> {
+      it.skip("bubbles up error from service for invalid partitionId", async function () {
```

Comment:
This was skipped by mistake I believe, the skips in this file is because the tracing assertion is not working at the moment and I left a comment to that effect.